### DISCUSSION TITLE
提供类似 GitHub 的 React with emoji 功能

### DISCUSSION THREAD

**Discussion by wan92hen** [halo-dev/halo#6645] [Ideas]
### 你当前使用的版本

2.0.2

### 描述一下此特性

类似点赞/喜欢功能，但是可以使用各种不同的 Emoji 表情对文章进行标记。
![image](https://user-images.githubusercontent.com/27671436/208389807-323029a7-6d65-42cc-8075-2a6ca8238b5a.png)

**Comment by ruibaby** [comment]
当前不考虑在核心中实现，后续如果有需求，建议通过插件实现。
